cypress/support/step_definitions/testSetup.js

```diff
@@ -93,13 +93,14 @@ function destroyAppInstance(testType) {
       params,
       additionalParams
     );
+    cy.log('Lifecycle close intent: ' + JSON.stringify(intentMessage));
 
```

Comment:
log in line 96 can be changed to 

Sending Lifecycle close intent to unload app